middle_end/closure/closure.ml

```diff
@@ -1652,7 +1659,9 @@ let intro ~backend ~size lam =
   in
   let opaque =
     !Clflags.opaque
-    || Env.is_imported_opaque (Compilenv.current_unit_name ())
+    || Env.is_imported_opaque
+         (Compilation_unit.get_current_exn ()
+          |> Compilation_unit.full_path_as_string)
```

Comment:
Not sure this is correct -- the old code used `Compilenv.current_unit_name` which just read the `ui_name` field, which I thought didn't have the prefix?

Reply:
Changed to use just the name.